{{ cookiecutter.package_name|slugify }}/pyproject.toml

```diff
@@ -58,10 +58,11 @@ uvicorn = { extras = ["standard"], version = "^0.16.0" }
 # https://python-poetry.org/docs/master/managing-dependencies/
 # TODO: Split in `tool.poetry.group.dev` and `tool.poetry.group.test` when Poetry 1.2.0 is released.
 [tool.poetry.dev-dependencies]
-cruft = "^2.10.1"
+bandit = "1.7.2"  # TODO: remove this line once this is solved: https://github.com/tylerwince/flake8-bandit/issues/21
```

Comment:
Apparently this has been fixed now, version 3.0.0 has been released 4 hours ago. Could you update this PR with that new version?

Reply:
Just tested the new version to be sure and in deed, it works :). Explicit bandit dependency removed and upgraded flake8-bandit dependency to 3.0.0.
Should be good to merge :)